src/main/java/uk/gov/hmcts/reform/idam/client/models/test/UserGroup.java

```diff
@@ -6,5 +6,5 @@
 @Getter
 @AllArgsConstructor
 public class UserGroup {
-    private final String code;
```

Comment:
same here?

Reply:
I pushed this WIP, not finished yet, and pushed in error also.